binding.cc

```diff
@@ -655,7 +655,7 @@ namespace zmq {
       return Nan::ThrowError("Must pass an option");
     if (!info[0]->IsNumber())
       return Nan::ThrowTypeError("Option must be an integer");
-    int64_t option = Nan::To<int64_t>(info[0]).FromJust();
+    int option = Nan::To<int>(info[0]).FromJust();
```

Comment:
There are a few int64 options. Hopefully [The docs](http://api.zeromq.org/4-1:zmq-getsockopt) cover it. pyzmq handles it by [keeping track](https://github.com/zeromq/pyzmq/blob/v16.0.0/zmq/backend/cython/socket.pyx#L434) of a list of options that should be int64, int, bytes.